labonneboite/common/offers.py

```diff
@@ -49,7 +49,14 @@ def get_offices(self):
 
         office_key_to_offers = defaultdict(list)
         for offer in offers:
-            if 'entreprise' in offer and 'siret' in offer['entreprise']:
+            offer_is_valid = (
+                'entreprise' in offer
+                and 'siret' in offer['entreprise']
+                and 'lieuTravail' in offer
+                and 'latitude' in offer['lieuTravail']
+                and 'longitude' in offer['lieuTravail']
+            )
+            if offer_is_valid:
```

Comment:
Petite remarque de style : j'aime bien `all()` et `any()`en Python :
https://docs.python.org/3/library/functions.html#all

Reply:
Brilliant! Will do.

Reply:
ou pas en fait! Cela casserait, car certaines de mes clauses cassent si la clause précédente est False.

Avantage de `and` vs `all`...

Je garde mes `and`